javascripts/tagmanager.js

```diff
@@ -953,6 +969,64 @@
                           callback(event, 'right');
                         }
                     }, true)
+                },
+                shouldElementBeMasked: function (element) {
```

Comment:
The logic here should cover all the cases

Reply:
@AltamashShaikh Could you please elaborate? This function accepts an element and returns whether or not the element needs to be masked. What else should be be doing?